plugin/js/showpass-calendar.js

```diff
@@ -425,7 +428,8 @@
 
             if (venue) {
 
-    			let url = "https://www.showpass.com/api/public/events/?venue__in=" + venue + "&page_size=100&starts_on__gte=" + startMonth;
+    			let url = "https://www.showpass.com/api/public/events/?venue__in=" + venue +
+                    "&page_size=100&starts_on__lte=" + endOfMonth + "&ends_on__gte=" + startOfMonth;
 
```

Comment:
Shouldn't these dates be reversed? I think this query is saying getting events where it starts on end of month and ends on start of month

Reply:
Yea it's a little confusing haha but it's saying
get events with a starts on date less than or equal to the end of the month
and an ends on date greater or equal to the start of the month

Meaning the event starts on needs to be before the end of the month they are viewing and have an ends on after the start of the month they are viewing, guaranteeing that past and future events won't be queried.

GIVEN event 6/20/2021 - 6/21/2021
WHEN viewing July 2021
THEN event won't be returned (only meets the `starts_on_lte` condition)

GIVEN event 7/10/2022 - 7/13/2022
WHEN viewing July 2021
THEN event won't be returned (only meets the `ends_on_gte` condition)

GIVEN event 6/20/2021 - 7/10/2021
WHEN viewing July 2021
THEN event won't be returned (meets both conditions but the starts on month is not equal to July so will get filtered out by the logic)